src/app-layout/__integ__/app-layout-split-panel.test.ts

```diff
@@ -69,239 +69,272 @@ class AppLayoutSplitViewPage extends BasePageObject {
+      setupTest(async page => {
+        await page.openPanel();
+        if (position === 'side') {
+          await page.switchPosition('side');
+          await page.keys(['Shift', 'Tab', 'Shift']);
+        }
+        await expect(page.isFocused(wrapper.findSplitPanel().findSlider().toSelector())).resolves.toBe(true);
+        // send each keystroke as individual command to allow UI re-rendering between keys
+        for (const key of Array(30).fill(repeatKey)) {
+          await page.keys(key);
+        }
+        await expect(page.getSplitPanelSliderValue()).resolves.toBe(expectedValue);
+      })()
+  );
+
+  test(
+    'renders with initial side position',
+    useBrowser(async browser => {
+      const page = new AppLayoutSplitViewPage(browser);
+      await page.setWindowSize(viewports.desktop);
+      await browser.url(`#/light/app-layout/with-split-panel?visualRefresh=false&splitPanelPosition=side`);
+      await page.waitForVisible(wrapper.findContentRegion().toSelector());
+      await page.openPanel();
+      await expect(page.getPanelPosition()).resolves.toEqual('side');
+    })
+  );
+  test(
+    'switches to bottom position when screen resizes to mobile',
+    setupTest(async page => {
+      await page.openPanel();
+      await page.switchPosition('side');
+      await expect(page.getPanelPosition()).resolves.toEqual('side');
+
+      await page.setWindowSize(viewports.mobile);
+      await expect(page.getPanelPosition()).resolves.toEqual('bottom');
+      // in VR design, split panel keeps same size as it was open on the side
+      const expectedBottomOffset = theme === 'visual-refresh' ? '440px' : '160px';
+      await expect(page.getContentOffsetBottom(theme)).resolves.toEqual(expectedBottomOffset);
+    })
+  );
   test(
-    `should not allow resize split panel beyond min and max limits (side position) (${name})`,
+    'switches to bottom position when screen is too narrow and restores back on resize',
     setupTest(async page => {
       await page.openPanel();
       await page.switchPosition('side');
-      const { width } = await page.getWindowSize();
-      await page.dragResizerTo({ x: width, y: 0 });
-      expect((await page.getSplitPanelSize()).width).toEqual(280);
+      await expect(page.getPanelPosition()).resolves.toEqual('side');
+
+      // narrow enough to force bottom position, but still not mobile
+      await page.setWindowSize({ ...viewports.desktop, width: 820 });
+      await expect(page.isExisting(`.${mobileStyles['mobile-bar']}`)).resolves.toBe(false);
+      await expect(page.getPanelPosition()).resolves.toEqual('bottom');
+
+      await page.setWindowSize(viewports.desktop);
+      await expect(page.getPanelPosition()).resolves.toEqual('side');
+    })
+  );
+
+  test(
+    'switches to bottom position when when tools panel opens and available space is too small',
+    setupTest(async page => {
+      await page.setWindowSize({ ...viewports.desktop, width: 1100 });
```

Comment:
Needed to update this value to find a breakpoint where "forced to bottom" behavior happens on all design variants